README.md

```diff
@@ -34,3 +34,7 @@ In the future Hive will support more cloud providers.
   * [Hive Architecture](./docs/architecture.md)
   * [SyncSet](./docs/syncset.md)
   * [SyncIdentityProvider](./docs/syncidentityprovider.md)
+  
+# Office Hours
+
+Hive team hosts virtual office hours every third Friday at 2 PM EST, open for Hive users. Please reach out for an invite. 
```

Comment:
Can we add a Google meet link? (we wouldn't want to use our normal scrum line but a separate link would be good)

Reply:
Added